documentation/RLC-test-quickstart.md

```diff
@@ -0,0 +1,234 @@
+# Table of contents
+
+- [Table of contents](#table-of-contents)
+- [Overview](#overview)
+- [Prerequisites](#prerequisites)
+- [How to Run Test](#how-to-run-test)
+  - [Test strucure](#test-strucure)
+  - [Fail to run tests in the first time](#fail-to-run-tests-in-the-first-time)
+  - [Running tests in playback mode](#running-tests-in-playback-mode)
+- [Writing New Tests](#writing-new-tests)
+  - [Example: Basic Azure service interaction and recording](#example-basic-azure-service-interaction-and-recording)
+
+# Overview
+
+This page is to help you write and run tests for Azure Javascript SDK. The Azure SDK test framework uses the [`test-recorder`](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/README.md) library, which in turn rests upon on a HTTP recording system ([testproxy](https://github.com/Azure/azure-sdk-tools/tree/main/tools/test-proxy)) that enables tests dependent on network interaction to be run offline.
```

Comment:
Is this for RLC only or for js sdks?  From the name of this file, seems this is for RLC only.

Reply:
Most of them are suitable for both RLC and HLC except the test structure part. HLC has slightly different structure with RLC.